### PR TITLE
add logout url to configuration file

### PR DESCRIPTION
**Pull request by jesuspg**
@flopezag @jframos

### CODE REVIEW

File: dashboard/config/dashboard.yml [hunk 1]
```diff
@@ -41,7 +41,8 @@ cbroker:
 idm:
   clientId: ''				# Id after registering app in IdM
   clientSecret: ''			# Secret key of the registered app
-  url: ''				# IdM's authentication URL
+  url: ''				    # IdM's authentication URL
+  logoutURL: ''             # Idm's logout URL
```
Comment (flopezag) [inline dashboard/config/dashboard.yml]: Why I see different tabular levels for these comments?

**Comment by flopezag** [comment]
LGTM

**Comment by jframos** [comment]
LGTM

File: dashboard/lib/routes/index.js [hunk 2]
```diff
@@ -62,7 +63,7 @@ function get_index (req, res) {
 
                 logger.debug('before render: %s', JSON.stringify(regions));
                 console.log({name: userinfo.displayName, regions: regions, role: req.session.role});
-                res.render('logged', {name: userinfo.displayName, regions: regions, role: req.session.role});
+                res.render('logged', {name: userinfo.displayName, regions: regions, role: req.session.role,logout_url: config.idm.logoutURL});
```
Comment (flopezag) [inline dashboard/lib/routes/index.js:66]: mmm, this line has more than 120 characters and I suggest to separate the "req.session.role,logout_url" in "req.session.role, logout_url..."

File: dashboard/lib/routes/index.js [hunk 3]
```diff
@@ -71,7 +72,7 @@ function get_index (req, res) {
 
         }
         else {
-            res.render('index', {name: 'sign in', regions: regions, role: req.session.role});
+            res.render('index', {name: 'sign in', regions: regions, role: req.session.role, logout_url: config.idm.logoutURL});
```
Comment (flopezag) [inline dashboard/lib/routes/index.js:75]: 128 characters

**Pull request merged**